docs/user/set-up/install-agent-binary.md

```diff
@@ -0,0 +1,32 @@
+---
+aliases:
+- /docs/agent/latest/set-up/install-agent-binary/
+title: Install the Grafana Agent binary
+weight: 140
+---
+
+## Install the Grafana Agent binary
+
+Install Grafana Agent and get it up and running using a binary file. 
+
+### Overview
+Binary executables are provided for the most common operating systems. Choose from the Assets list on the Releases page that matches your operating system. 
+
+ppc64le builds are currently considered secondary release targets and do not have the same level of support and testing as other platforms.
+
+### Steps
```

Comment:
Are we moving to this convention to introduce steps rather than the stem sentence? Has this been updated in the style guide and communicated to the writers?

Reply:
see comment below. let's discuss in the team meeting?